src/internal/hooks/use-dynamic-overlap/__tests__/use-dynamic-overlap.test.tsx

```diff
@@ -1,10 +1,10 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Moved this file from `src/app-layout/hooks` into `src/internal/hooks` because it is in fact a cross-component API